### PR TITLE
feat: Announcement bar added for `Slack invitation`

### PR DESCRIPTION
**Pull request by iamayushdas**
Changes:
added a feature - announcement bar whih will notify users for new releases and after some days we can use next announcement for join slack channel

Screenshots of the change:
announcemnet

### CODE REVIEW

**Comment by iamayushdas** [comment]
cc - @juzhiyuan @Yiyiyimu @LiteSun

File: website/docusaurus.config.js [hunk 1]
```diff
@@ -415,6 +415,18 @@ module.exports = {
       copyright:
         "Copyright © 2019-2021 The Apache Software Foundation. Apache APISIX, APISIX™, Apache, the Apache feather logo, and the Apache APISIX project logo are either registered trademarks or trademarks of the Apache Software Foundation.",
     },
+    announcementBar: {
+      id: 'apisix-2.6-Release',
```
Comment (juzhiyuan) [inline website/docusaurus.config.js]: No, 2.6 is in progress and VOTE.

Comment (iamayushdas) [reply website/docusaurus.config.js]: its for after release

File: website/docusaurus.config.js [hunk 2]
```diff
@@ -415,6 +415,18 @@ module.exports = {
...
+      content:
+        '➡️ Apache APISIX v2.6.0 released, users can view Changelogs <a target="_blank" rel="noopener noreferrer" href="https://github.com/apache/apisix/blob/release/2.6/CHANGELOG.md">APISIX 2.6.0 release</a>! 🔄',
```
Comment (Yiyiyimu) [inline website/docusaurus.config.js]:
```suggestion
        '➡️ Apache APISIX v2.6.0 released, users can view Changelogs <a target="_blank" rel="noopener noreferrer" href="https://github.com/apache/apisix/blob/2.6/CHANGELOG.md#260">APISIX 2.6.0 release</a>! 🔄',
```

**Comment by Yiyiyimu** [comment]
White background is not good for daylight theme

**Comment by iamayushdas** [comment]
> White background is not good for daylight theme

Okay, will do something with that. Thanks for your opinion

**Comment by iamayushdas** [comment]
scrnli_25_05_2021_13-40-57
scrnli_25_05_2021_13-41-19

**Comment by iamayushdas** [comment]
> White background is not good for daylight theme

have a look now

**Comment by iamayushdas** [comment]
@juzhiyuan @Yiyiyimu apisix 2.6 has been released can we now add this announcement bar?

File: website/docusaurus.config.js [hunk 3]
```diff
@@ -415,6 +415,22 @@ module.exports = {
...
+    announcementBar: {
+      id: 'apisix-2.6-Release',
+      backgroundColor: "#e8433e",
+      textColor: 'white',
+      content:
+        '➡️ Apache APISIX v2.6.0 released, users can view Changelogs <a target="_blank" rel="noopener noreferrer" href="https://github.com/apache/apisix/blob/2.6/CHANGELOG.md#260">APISIX 2.6.0 release</a>! 🔄',
+    },
+    /*
+    announcementBar: {
+      id: 'query',
```
Comment (juzhiyuan) [inline website/docusaurus.config.js:419]: Do we need to keep this?

Comment (iamayushdas) [reply website/docusaurus.config.js:419]: Yes that will make easier for us to change the announcement just by un commenting it, docusaurus and many websites uses the same strategy

**Comment by iamayushdas** [comment]
hey @Yiyiyimu is it ready for approval?

**Comment by Yiyiyimu** [comment]
Sorry I just realized I'm not sure about the need of adding the announcement bar. People who enter our website would always check the latest version of APISIX so it seems could not be of much help. I think maybe it's a good to keep our welcome page as simple as possible

**Comment by iamayushdas** [comment]
What if we add slack invitation to the announcement bar, like many of the docusaurus based websites does?

**Comment by Yiyiyimu** [comment]
> What if we add slack invitation to the announcement bar, like many of the docusaurus based websites does?

That's a good idea, could you raise some examples

**Comment by iamayushdas** [comment]
@Yiyiyimu see examples
scrnli_02_06_2021_08-55-04
scrnli_02_06_2021_08-56-25
scrnli_02_06_2021_08-56-48

how it will look on our website
scrnli_02_06_2021_11-03-50

**Comment by Yiyiyimu** [comment]
That's pretty cool! Need confirmation from @juzhiyuan

**Review by KishaniKandasamy — APPROVED** [review]

**Comment by iamayushdas** [comment]
@juzhiyuan could you please drop your review here?

**Review by juzhiyuan — APPROVED** [review]

**Review by liuxiran — APPROVED** [review]

**Pull request merged**